h/migrations/versions/64039842150a_add_user_comms_opt_in_column.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
```

Comment:
```suggestion
```

These aren't needed in Python 3 (we should update [the template](https://github.com/hypothesis/h/blob/master/h/migrations/script.py.mako))